### PR TITLE
Support text color

### PR DESCRIPTION
**Pull request by yangbin**

### CODE REVIEW

**Comment by zz85** [comment]
:+1:

File: jx/ui/html/HTMLRenderer.js [hunk 1]
```diff
@@ -35,17 +35,12 @@ HTMLRenderer.prototype.onDirty = function(dirtylist) {
 		if (this.root_dom)
 			this.parent_dom.appendChild(this.root_dom);
 	}
+	console.log(this.renderlist);
```
Comment (jieyanhuang) [inline jx/ui/html/HTMLRenderer.js]: should this be here?

**Comment by jieyanhuang** [comment]
lgtm :+1:

**Comment by zz85** [comment]
could make `background` -> `backgroundColor` to be more specific

**Comment by yangbin** [comment]
Will do that in a separate PR

**Pull request merged**